src/containers/edit-profile/professional-info-tab/professional-category/ProfessionalCategory.styles.ts

```diff
@@ -0,0 +1,85 @@
+import { TypographyVariantEnum } from '~/types'
+import palette from '~/styles/app-theme/app.pallete'
+
+const { Body2, Overline, Caption } = TypographyVariantEnum
+
+export const styles = {
+  root: (isActivated: boolean) => ({
+    border: `1px solid ${
+      isActivated ? palette.primary[300] : palette.primary[100]
+    }`,
+    display: 'flex',
+    flexDirection: 'column',
+    gap: 2,
+    borderRadius: 1,
+    px: 2,
+    py: 4
+  }),
+  createBtnContainer: {
+    my: 3
+  },
+  toolbar: {
+    root: {
+      display: 'flex',
+      justifyContent: 'space-between'
```

Comment:
We have to add `alignItems: 'center'` to make delete buttons round, as them look oval now
<img width="70" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/110097862/680b8502-fed0-4eab-b15d-5eb8d8b0da71">

```suggestion
      justifyContent: 'space-between',
      alignItems: 'center'
```

Reply:
Thank a lot, really icon has incorrect shape! I used your suggested change